src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState, ChangeEventHandler, Fragment, FC } from 'react'
+import { useHistory } from 'react-router-dom'
+import { useDispatch } from 'react-redux'
+import { Button, Container, Paper, Grid } from '@material-ui/core'
+
+import { Typography, Dropdown } from '../../components'
+import { startTest } from '../../redux'
+import { homeStyles } from './Home.styles'
+import { levels } from './Home.constant'
+
+interface HomeProps {}
+
+const Home: FC<HomeProps> = () => {
+  const classes = homeStyles()
+  const dispatch = useDispatch()
+  const history = useHistory()
+  const [level, setLevel] = useState<string>('')
+
+  const handleChange: ChangeEventHandler<HTMLInputElement> = ({
+    target: { value },
+  }) => {
+    setLevel(value)
+    dispatch(startTest(value))
+  }
+
+  const handleStartClick = () => {
+    history.push('typing-test')
+  }
+
+  return (
+    <Fragment>
```

Comment:
Think we don't need fragment here as well. Also it would be a good idea to be consistent regarding using `<Fragment>` or `</>`. Should use one and stick with it.